CONTRIBUTING.md

```diff
@@ -77,11 +79,14 @@ the [development mailing list](http://dev.list.galaxyproject.org/).
   request](https://help.github.com/articles/creating-a-pull-request/)
   with these changes. You pull request message ideally should include:
 
-   * A description of why the change should be made.
+   * A description of why the changes should be made.
 
-   * A description of implementation of the change.
+   * A description of the implementation of the changes.
 
-   * A description of how to test the change.
+   * A description of how to test the changes.
+
+* The pull request should pass all the continuous integration tests which are
```

Comment:
@jxtx I'm not a native speaker, so please suggest a rephrase. Anyway there are other 16 `should` instances in this same document...